packages/popup/src/App.js

```diff
@@ -5,7 +5,6 @@ import {HashRouter as Router, Redirect, Route, Switch} from 'react-router-dom'
 import {useIsLocked, useIsZeroGroup, usePendingAuthReq} from './hooks/useApi'
 import {ProtectedRoute} from './components'
 import {ROUTES} from './constants'
-import './index.css'
```

Comment:
When do the components css separation

Reply:
there's a dup one in app.js

Reply:
https://github.com/Conflux-Chain/helios/blob/2aa0159e4402c4b66cc4421b546bde98f37ac036/packages/popup/src/index.js#L3